blocks/search/autosuggest.js

```diff
@@ -1,25 +1,28 @@
-import { createElement } from '../../scripts/common.js';
-import { autosuggestQuery, fetchData } from './search-api.js';
+import { createElement, getLocale } from '../../scripts/common.js';
+import { autosuggestQuery, fetchData, TENANT } from '../../scripts/search-api.js';
 
 const autoSuggestClass = 'autosuggest-results-item-highlighted';
 
 export function fetchAutosuggest(term, autosuggestEle, rowEle, func) {
   const fragmentRange = document.createRange();
+  const language = getLocale();
```

Comment:
I think you should rename these variables to:
`const locale = getLocale();`
`const language = language.split('-')[0].toUpperCase();`

Reply:
DONE